### ISSUE TITLE
All pages: Sub-menu navigation overlaps content when content is reflowed

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#147]
URL: https://rsams.dev.cloud.openplus.ca/en
User Agent: Chrome (v99), Firefox (v98)
Language: Both

Issue Description: When the content is reflowed, the sub-menu navigation overlaps content.
![image](https://user-images.githubusercontent.com/88776404/161259476-104a65a1-97c6-4d85-b4b7-1e2460689e02.png)

Steps to reproduce:

1. Navigate to the url above. Or Page name for the test is: 'Recalls and Safety Alerts Help Guide' in case it needs to be searched on for next test.
2. Set the desktop browser to 1280(W) X 1024(H) and zoom in to 400% (Firefox -> Web Developer settings ->Resize Window to 1280X1024).

Expected results: When content is zoomed up to 400% the content should be presented without loss of information or text overlap.
